package.json

```diff
@@ -86,6 +89,7 @@
     "eslint-plugin-github": "4.1.1",
     "eslint-plugin-jest": "22.20.0",
     "eslint-plugin-jsx-a11y": "6.2.3",
+    "eslint-plugin-prettier": "3.3.1",
```

Comment:
Was this intentional?

Reply:
Yeah. Linting failed on this branch without this dependency. It should automatically be installed with `eslint-plugin-github` but for some reason, it wasn't. I'm not too worried about leaving this in though because it's just a dev dependency.